diff --git a/modules/juce_audio_processors/utilities/juce_VST2ClientExtensions.h b/modules/juce_audio_processors/utilities/juce_VST2ClientExtensions.h
index f29fcbc..b635c08 100644
--- a/modules/juce_audio_processors/utilities/juce_VST2ClientExtensions.h
+++ b/modules/juce_audio_processors/utilities/juce_VST2ClientExtensions.h
@@ -76,6 +76,12 @@ struct VST2ClientExtensions
         You can use the supplied function to query the VST host.
     */
     virtual void handleVstHostCallbackAvailable (std::function<VstHostCallbackType>&& callback);
+
+    /** This is called if the host is REAPER and provides API functions.
+        The function pointer can be used to get REAPER API functions by name.
+        @param reaperGetFunc A function pointer to get REAPER API functions
+    */
+    virtual void handleReaperApi (void* (*reaperGetFunc)(const char*)) {}
 };
 
 using VSTCallbackHandler [[deprecated ("replace with VST2ClientExtensions")]] = VST2ClientExtensions;
